components/ViewBills/ViewBills.js

```diff
@@ -41,6 +41,19 @@ const BillRow = props => {
     ) : (
       <>{bill.PrimarySponsor ? bill.PrimarySponsor.Name : null}</>
     )
+
+  const CommitteeComponent = fullBill.currentCommittee ? (
```

Comment:
Since this is just jsx and not a component we call, let's rename this to committeeCell or something (first letter lowercase)